vllm/engine/metrics.py

```diff
@@ -24,64 +30,63 @@
 # begin-metrics-definitions
 class Metrics:
     labelname_finish_reason = "finished_reason"
+    _base_library = prometheus_client
 
     def __init__(self, labelnames: List[str], max_model_len: int):
         # Unregister any existing vLLM collectors
-        for collector in list(REGISTRY._collector_to_names):
-            if hasattr(collector, "_name") and "vllm" in collector._name:
-                REGISTRY.unregister(collector)
+        self._unregister_vllm_metrics()
 
         # Config Information
-        self.info_cache_config = Info(
+        self.info_cache_config = prometheus_client.Info(
```

Comment:
```suggestion
        self.info_cache_config = self._base_library.Info(
```

Reply:
Ah, Info is actually exclusive to prometheus and does not have a counterpart in ray.utils.metric

Reply:
Reverted by [be00080](https://github.com/vllm-project/vllm/pull/5925/commits/be0008051d06e24b624277f3ad3eaa7081161544)